.gitignore

```diff
@@ -77,3 +77,6 @@ doc/xml/
 
 # man pages
 *.1.gz
+
+# VS
+.vscode/
```

Comment:
empty vscode settings ?

Reply:
It simply ignores the entire `.vscode/` dir.

Reply:
Ah ok, you are coding in vscode have your own settings and don't want to be bothered by their contents, got it. Unrelated to the proposed changed but doesn't hurt, ok